pkg/apis/fleet.cattle.io/v1alpha1/gitrepo_types.go

```diff
@@ -129,6 +129,9 @@ type GitRepoSpec struct {
 
 	// CorrectDrift specifies how drift correction should work.
 	CorrectDrift *CorrectDrift `json:"correctDrift,omitempty"`
+
+	// Disables git polling. When enabled only webhooks will be used.
+	DisableGitPolling bool `json:"disableGitPolling,omitempty"`
```

Comment:
Could this be shorted to `DisablePolling` as it doesn't do any other form of polling?